mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreAttachmentManager.java

```diff
@@ -53,6 +53,11 @@ public StoreAttachmentManager(AttachmentMapperFactory attachmentMapperFactory, M
     }
 
     @Override
+    public boolean exists(AttachmentId attachmentId, MailboxSession session) throws MailboxException {
```

Comment:
no test?

Reply:
There is no existing test for AttachmentManager, only tests on mapper.

As testing this might require either:
 - Creating generic tests (AbstractAttachmentManagerTests blah)
 - Relying on mocking (there is a little test with mocking, but I really don't like mocking)

So I would go for the first one, if you realy want it.

Reply:
no that's ok, manager is just wiring so we can go without these tests